appium/lib/screen-object/accessors/element.rb

```diff
@@ -37,88 +35,180 @@ def driver
       end
 
       def click
-          element.click
+        element.click
       end
 
       def value
-          element.value
+        element.value
       end
 
       def exists?
+        driver.no_wait
         begin
-          element.displayed?
-        rescue
+          driver.set_wait if element.displayed?
+          true
+        rescue StandardError
           false
         end
       end
 
       def element
-        driver.find_element(:"#{locator[0]}",locator[1])
+        driver.find_element(:"#{locator[0]}", locator[1].to_s)
       end
 
       def elements
-        driver.find_elements(:"#{locator[0]}",locator[1])
+        driver.find_elements(:"#{locator[0]}", locator[1].to_s)
       end
 
       def element_attributes
         %w[name resource-id value text]
       end
 
+      def rect
+        my_element = element.rect
+        opt = {}
```

Comment:
This can be simplified to not use an intermediate variable `opt` as such:
```ruby
  {
    start_x: my_element.x
    end_x: my_element.x + my_element.width
    start_y: my_element.y
    end_y: my_element.y + my_element.height
    height: my_element.height
    width: my_element.width
  }
```